### ISSUE TITLE
Address duplicate UI bug

### ISSUE THREAD

**Issue by wpscholar** [newfold-labs/wp-module-performance#10]
<img width="727" alt="MicrosoftTeams-image" src="https://github.com/newfold-labs/wp-module-performance/assets/890951/3601858a-19f0-44c4-a0e9-d23c78b38864">

**Description:**
When a brand plugin is active and the EPC MU plugin is also active, there is a duplicate UI on the settings page regarding caching options. Ideally, there should only be one set of options. However, there may be some differences between the EPC options and the module options.

Reported here: https://core.trac.wordpress.org/ticket/59210

References where we check for the EPC MU plugin: https://github.com/search?q=repo%3Anewfold-labs%2Fwp-module-performance%20hasMustUsePlugin&type=code

**Comment by arunshenoy99** [comment]
@geckod22 @AleTorrisi Can you check if we still need to do this? If I’m not wrong, we already fixed it. If that’s the case, I’ll go ahead and close it out. Thanks!

**Comment by geckod22** [comment]
@arunshenoy99  Yes it should be fixed time ago, we have removed that part from settings.

**Comment by arunshenoy99** [comment]
> [@arunshenoy99](https://github.com/arunshenoy99) Yes it should be fixed time ago, we have removed that part from settings.

Perfect, thanks!

**Comment by AleTorrisi** [comment]
Yes I confirm it is no longer available
